depthai.py

```diff
@@ -349,7 +361,8 @@ def show_landmarks_recognition(entries_prev, frame):
     'depth':
     {
         'calibration_file': consts.resource_paths.calib_fpath,
-        'padding_factor': 0.3
+        'padding_factor': 0.3,
+        'depth_limit_m': 10.0, # In meters, for filtering purpose during x,y,z calc
```

Comment:
For now just set here. Should we add an option for it, or just think about a good default value?